website/docs/cloud-docs/api-docs/explorer.mdx

```diff
@@ -0,0 +1,459 @@
+        "provider-count": 0,
+        "providers": null,
+        "resources-drifted": 0,
+        "resources-undrifted": 0,
+        "state-version-terraform-version": "1.5.7",
+        "vcs-repo-identifier": null,
+        "workspace-created-at": "2023-10-17T21:56:45.570+00:00",
+        "workspace-name": "payments-service",
+        "workspace-terraform-version": "1.5.7",
+        "workspace-updated-at": "2023-12-08T19:58:15.513+00:00"
+      },
+      "id": "ws-j2sAWeRxuo1b5HYf",
+      "type": "visibility-workspace"
+    },
+    {
+      "attributes": {
+        "all-checks-succeeded": true,
+        "checks-errored": 0,
+        "checks-failed": 0,
+        "checks-passed": 0,
+        "checks-unknown": 0,
+        "current-run-applied-at": "2023-08-18T16:24:59.000+00:00",
+        "current-run-external-id": "run-9MmJaoQTvDCh5wUa",
+        "current-run-status": "applied",
+        "drifted": true,
+        "external-id": "ws-igUVNQSYVXRkhYuo",
+        "module-count": 0,
+        "modules": null,
+        "organization-name": "acme-corp",
+        "project-external-id": "prj-uU2xNqGeG86U9WgT",
+        "project-name": "web",
+        "provider-count": 0,
+        "providers": null,
+        "resources-drifted": 3,
+        "resources-undrifted": 3,
+        "state-version-terraform-version": "1.4.5",
+        "vcs-repo-identifier": "acmecorp/api",
+        "workspace-created-at": "2023-04-25T17:09:14.960+00:00",
+        "workspace-name": "api-service",
+        "workspace-terraform-version": "1.4.5",
+        "workspace-updated-at": "2023-12-08T20:43:16.779+00:00"
+      },
+      "id": "ws-igUNVQSYXVRkhYuo",
+      "type": "visibility-workspace"
+    }
+  ],
+  "links": {
+    "self": "https://app.terraform.io/api/v2/organizations/acme-corp/explorer?page%5Bnumber%5D=1&page%5Bsize%5D=20&type=workspaces",
+    "first": "https://app.terraform.io/api/v2/organizations/acme-corp/explorer?page%5Bnumber%5D=1&page%5Bsize%5D=20&type=workspaces",
+    "last": "https://app.terraform.io/api/v2/organizations/acme-corp/explorer?page%5Bnumber%5D=1&page%5Bsize%5D=20&type=workspaces",
+    "prev": null,
+    "next": null
+  },
+  "meta": {
+    "pagination": {
+      "current-page": 1,
+      "page-size": 20,
+      "next-page": null,
+      "prev-page": null,
+      "total-pages": 1,
+      "total-count": 2
+    }
+  }
+}
+
+```
+
+## Export data as CSV
+
+`GET /organizations/:organization_name/explorer/export/csv`
+
+| Parameter            | Description |
+| -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `:organization_name` | The name of the organization to query. The organization must already exist in the system, and the user must have permissions to read the workspaces and projects within it. |
+
+This endpoint can be used to download a full, unpaged export of the query results
```

Comment:
Can we also specify that the filters and sorts will also be applied to the export result? 
but the pagination will not (that's covered here)